__tests__/app.test.js

```diff
@@ -114,4 +113,42 @@ describe('/api/articles', () => {
             expect(body.articles).toBeSortedBy('created_at', {descending: true})
         })
     })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 responds with an array of comments objects', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments).toHaveLength(11)
+            const commentFormat = expect.objectContaining({
+                comment_id: expect.any(Number),
+                article_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+            })
+            body.comments.forEach((comment) => {
+                expect(comment).toEqual(commentFormat);
+            })
+        })
+    })
+    test('GET:200 comments are ordered by date by default and in descending order', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments).toBeSortedBy('created_at', {descending: true})
+        })
+    })
+    test('GET:404 sends an appropriate status and error message when given an id that has no comments', () => {
+        return request(app)
+        .get('/api/articles/8/comments')
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe('no comments found');
+        });
```

Comment:
In this case if the article exists we would want to return an empty array and a 200 - there's no comments _yet_ but the article does exist. If you imagine what this would look like on the front end you might get a 'be the first to add a comment' prompt rather than an error. 

We do however want to return a 404 if the article doesn't exist - for that I would use a really high number to be sure. 

To check the above you'll need to check that the article exists in the database by doing a separate check. This could be done when you get empty rows or even before with a Promise.all

We also would want a 400 test for /api/articles/not-a-number/comments